gradle.properties

```diff
@@ -0,0 +1,6 @@
+//signing.keyId=<publicKey>
```

Comment:
Unless it's OK to publish these credentials publicly, I wouldn't keep them in a version controlled file. Would it be possible to have gradle.properties outside the project tree? Or if not - maybe keep a `*.sample` and add `gradle.properties` to `.gitignore`.